src/peppr/selector.py

```diff
@@ -129,3 +130,33 @@ def select(self, values: np.ndarray, smaller_is_better: bool) -> float:
             return np.nanmin(top_values)
         else:
             return np.nanmax(top_values)
+
+
+class RandomSelector(Selector):
+    """
+    Selector that returns the best value from `k` randomly chosen values.
+    Using this selector is equivalent to using the :class:`TopSelector` with
+    random confidence values.
+
+    Parameters
+    ----------
+    k : int
+        The best value is chosen from *k* randomly chosen predictions.
+    """
+
+    def __init__(self, k: int) -> None:
```

Comment:
I think it would be good to be able to provide an optional `seed` to the constructor that would be used to initialize a NumPy [`Generator`](https://numpy.org/doc/2.1/reference/random/generator.html#numpy.random.default_rng)

Reply:
Good point, addressed in ce196e11f45720f17896029fe8c6a60c153e7b7a!